### PR TITLE
Add Azure info to Cloud docs

### LINKED ISSUE #2008: Add information about Azure marketplace integration
Users will soon be able to sign up for InfluxDB Cloud through Azure Marketplace. We need to include it in the Cloud getting started doc and notes in the account management doc.

We should also document the Microsoft SSO functionality added to the Login/Sign Up form while we're at it.

##### Relevant URLs
- https://docs.influxdata.com/influxdb/cloud/get-started/#subscribe-through-a-cloud-provider
- https://docs.influxdata.com/influxdb/cloud/get-started/#subscribe-through-influxdata
- https://docs.influxdata.com/influxdb/cloud/account-management/billing/#access-billing-details

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #2008. @gunnaraasen, do you have the link for Cloud on Azure marketplace?

### CODE REVIEW

**Review by kelseiv — COMMENTED** [review]
Looks good - approved once we get the links in/verify setup steps!

File: content/influxdb/cloud/get-started.md [hunk 1]
```diff
@@ -32,29 +32,35 @@ Limits are designed to let you monitor 5-10 sensors, stacks or servers comfortab
     To subscribe to an InfluxDB Cloud **Free Plan** through InfluxData,
     go to [InfluxDB Cloud](https://cloud2.influxdata.com/).
 
-    - To use social sign-on, click **Google**. Note that Google social sign-on does not support email aliases.
+    - To use social sign-on, click  **Azure** or **Google**. Note that Google social sign-on does not support email aliases.
```
[the remark this answers is not in the document]
Comment (noramullen1) [reply content/influxdb/cloud/get-started.md:35]: @gunnaraasen do you know the answer to this?

[the remark this answers is not in the document]
Comment (gunnaraasen) [reply content/influxdb/cloud/get-started.md:35]: It does not support email aliases so it's correct as is.

**Comment by gunnaraasen** [comment]
Once it's live, the public URL for InfluxDB Cloud on Azure Marketplace should be https://azuremarketplace.microsoft.com/en-us/marketplace/apps/influxdata.influxdb-cloud.

File: content/influxdb/cloud/account-management/billing.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ products: [cloud]
     {{< nav-icon "account" >}}
 
 2. Do one of the following:
-    - If you subscribed to an InfluxDB Cloud plan through [**AWS Marketplace**](https://aws.amazon.com/marketplace/pp/B08234JZPS) or [**GCP Marketplace**](https://console.cloud.google.com/marketplace/details/influxdata-public/cloud2-gcp-marketplace-prod?pli=1), click the **AWS** or **GCP** link to access your billing and subscription information.
+    - If you subscribed to an InfluxDB Cloud plan through [**AWS Marketplace**](https://aws.amazon.com/marketplace/pp/B08234JZPS), [**Azure Marketplace**](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/influxdata.influxdb-cloud), or [**GCP Marketplace**](https://console.cloud.google.com/marketplace/details/influxdata-public/cloud2-gcp-marketplace-prod?pli=1), click the **AWS**, **Azure**, or **GCP** link to access your billing and subscription information.
```
Comment (noramullen1) [inline content/influxdb/cloud/account-management/billing.md:33]:
```suggestion
    - If you subscribed to an InfluxDB Cloud plan through [**AWS Marketplace**](https://aws.amazon.com/marketplace/pp/B08234JZPS), [**Azure Marketplace**](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/influxdata.influxdb-cloud), or [**GCP Marketplace**](https://console.cloud.google.com/marketplace/details/influxdata-public/cloud2-gcp-marketplace-prod?pli=1), click the **AWS**, **Microsoft Azure**, or **GCP** link to access your billing and subscription information.
```

File: content/influxdb/cloud/get-started.md [hunk 3]
```diff
@@ -32,29 +32,35 @@ Limits are designed to let you monitor 5-10 sensors, stacks or servers comfortab
     To subscribe to an InfluxDB Cloud **Free Plan** through InfluxData,
     go to [InfluxDB Cloud](https://cloud2.influxdata.com/).
 
-    - To use social sign-on, click **Google**. Note that Google social sign-on does not support email aliases.
+    - To use social sign-on, click  **Azure** or **Google**. Note that social sign-on does not support email aliases.
```
Comment (kelseiv) [inline content/influxdb/cloud/get-started.md:35]: **Google** or **Microsoft**

**Pull request merged**